Security/Data_Classification.mediawiki

```diff
@@ -47,106 +54,79 @@ information (e.g.: Google docs, text documents, presentations, attachments to em
-* Aggregate survey data about Mozilla employees.
-
-<span style="background-color: #ffd351; border-radius: .25em; color: #594300; display: inline-block; font-weight: bold;margin: .1em 0; min-width: 6em; padding: .05em .5em; text-transform: uppercase; text-align: center;">Mozilla Confidential - Specific Work Groups Only</span>
-
-* Employee's street address, SSNs, performance data.
-* Service passwords/credentials.
-* Bugzilla bugs with security or restricted flags.
-* Proprietary or protected information, code, libraries from Mozilla partners.
-* Contracts or legal documents.
-* Unannounced communication materials (dates, visuals, plans) for campaigns, product launches, etc.
-
-<span style="background-color: #d04437; border-radius: .25em; color: #ffffff; display: inline-block; font-weight: bold;margin: .1em 0; min-width: 6em; padding: .05em .5em; text-transform: uppercase; text-align: center;">Mozilla Confidential - Specific Individuals Only</span>
-
-* Firefox release signing keys.
-* Specific partner conversations.
-* Employee bank account information.
-* User/personal passwords/credentials.
-* [https://www.mozilla.org/en-US/about/governance/policies/participation/ Community Participation Guideline (CPG)] report data.
-
-= Help to label data in emails, gdocs, presentations, wiki, code, videos, etc. =
+= Deciding how to classify  =
 
 ''The list of examples of how to label data is not an exhaustive list and serves an an indication on how to ensure the data classification labels are clearly communicated.''
 
-There are always two people involved with exchanging Confidential information:
+There are always at least two people involved with exchanging confidential information:
 
-* The '''Discloser''' is the person who provides the information to the Recipient.
+* The '''Owner''' is the person who created the file. The '''Owner''' and '''Discloser''' are often the same person, but not always.
+* The '''Discloser''' is the person who provides/sends/shares the information to the Recipient.
 * The '''Recipient''' is the person who receives the information.
 
-== Keynote/Powerpoints, box.com, etc. ==
+When in doubt, it's always best to classify at the most specific classification.
+
+== All new documents, box.com, etc. ==
 
-'''Label''' every document with its appropriate classification at the top of the document. When possible, we recommend
-using the header feature of the document.
+'''Label''' every document with its appropriate classification at the top of the document if possible. When possible, we recommend
+using the header feature of the document. 
 
 == Google Apps ==
 
-'''Label''' every document (Docs, Sheets, Slides, Drawings, etc.) with its appropriate classification at the top of
-the document.
+'''Label''' every document (Docs, Sheets, Slides, Drawings, etc.) with its appropriate classification by utilzing the required label functionality and if its not possible, attempt to insert it into the body of the document itself.
```

Comment:
```suggestion
'''Label''' every document (Docs, Sheets, Slides, Drawings, etc.) with its appropriate classification by utilizing the required label functionality.
```

I think we can just tell folks to use the label since there aren't any Google Drife file types that don't support the badge label.

Reply:
Right, glad we experimented and confirmed yesterday, thumbs up to commit that change